package.json

```diff
@@ -34,7 +35,15 @@
     "test:visual:update": " node ./spec/visual-regression-tester.js test --updateGolden",
     "version": "gulp build release",
     "watch": "NODE_ENV=development nswatch",
-    "prettier:scss": "npx prettier --write ./src/stylesheets/**/*.scss"
+    "prettier:scss": "npx prettier --write ./src/stylesheets/**/*.scss",
+    "patternlab": "patternlab",
+    "pl:dev": "node ./node_modules/@pattern-lab/uikit-workshop/build-tools.js",
+    "build:uikit": "cross-env-shell PL_CONFIG_PATH='${INIT_CWD}/.patternlabrc.js' npm run build --prefix node_modules/@pattern-lab/uikit-workshop -- --patternlabrc '$PL_CONFIG_PATH'",
+    "pl:build": "patternlab build --config ./patternlab-config.json",
+    "pl:help": "patternlab --help",
+    "pl:install": "patternlab install --config ./patternlab-config.json",
+    "pl:serve": "patternlab serve --config ./patternlab-config.json -v",
+    "pl:version": "patternlab --version"
```

Comment:
Let's only include the tasks we need to build/watch the library.

Reply:
you probably don't need any of them since they are just alias for some patternlab CLI tasks.

from the docs
```
		build|compile [options]   Build the PatternLab. Optionally (re-)build only the patterns
		export                    Export a PatternLab patterns into a compressed format
		init [options]            Initialize a PatternLab project from scratch or import an edition and/or starterkit
	        install|add [options]     Installs Pattern Lab related modules like starterkits or plugins
		serve|browse [options]    Starts a server to inspect files in browser
```

but agree, we could/should only include the one we are leveraging if it helps to reduce confusion "pl:build"